### ISSUE TITLE
Wrong italic glyphs for turned m

### ISSUE THREAD

**Issue by dscorbett** [googlei18n/noto-fonts#1097]
### Font
NotoSerif-Italic.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSerif-unhinted.zip
Date: 2018-03-13

### Font version
Version 2.000;GOOG;noto-source:20170915:90ef993387c0

### Issue
U+026F and U+1D5A should be turned versions of U+006D. In Noto Serif (Bold) Italic, their top left corners are angular whereas the bottom right corner of U+006D is curved. Cf. the bottom left corner of U+1D1F, which is also curved.

### Character data
﻿﻿mɯᵚᴟ
U+006D LATIN SMALL LETTER M
U+026F LATIN SMALL LETTER TURNED M
U+1D5A MODIFIER LETTER SMALL TURNED M
U+1D1F LATIN SMALL LETTER SIDEWAYS TURNED M

### Screenshot
![﻿﻿mɯᵚᴟ](https://user-images.githubusercontent.com/1124347/37556087-2f8c8c6e-29c7-11e8-8cca-c3d1d5b89d1b.png)

**Comment by moyogo** [comment]
While U+026F ɯ is called LATIN SMALL LETTER TURNED M, it doesn’t have to be exactly like a turned m. I’d say it is OK for it to be akin to u. That same applied to its superscript symbol U+1D5A ᵚ MODIFIER LETTER SMALL TURNED M.

**Comment by nizarsq** [comment]
Look like this behavior not specific to italic-bold, I can see it in italic regular weight. @marekjez86 does U+026F ɯ have to be exactly like a turned U+006D m. I think the current behavior make the layout consistent especially because it is serif font
<img width="602" alt="Screen Shot 2020-07-14 at 12 30 28 AM" src="https://user-images.githubusercontent.com/22158882/87397448-3898d700-c569-11ea-987a-776bdf27cc99.png">

**Comment by dscorbett** [comment]
> While U+026F ɯ is called LATIN SMALL LETTER TURNED M, it doesn’t have to be exactly like a turned m. I’d say it is OK for it to be akin to u. That same applied to its superscript symbol U+1D5A ᵚ MODIFIER LETTER SMALL TURNED M.

That is reasonable. Still, in that case, U+1D1F LATIN SMALL LETTER SIDEWAYS TURNED M ought to look like a sideways  U+026F LATIN SMALL LETTER TURNED M, whereas it now looks like a sideways turned  U+006D LATIN SMALL LETTER M. In other words, the bottom left end of U+1D1F should match the top left end of U+026F. So either way, the font has an inconsistency.

**Comment by moyogo** [comment]
> Still, in that case, U+1D1F LATIN SMALL LETTER SIDEWAYS TURNED M ought to look like a sideways U+026F LATIN SMALL LETTER TURNED M, whereas it now looks like a sideways turned U+006D LATIN SMALL LETTER M. In other words, the bottom left end of U+1D1F should match the top left end of U+026F. So either way, the font has an inconsistency.

Indeed, U+1D1F LATIN SMALL LETTER SIDEWAYS TURNED M is a rotated U+026F LATIN SMALL LETTER TURNED M.
The Uralic Phonetic Alphabet, in this case the variant with the vowel symbol ɯ used in Toivo Lehtisalo (1956) *Juraksamojedisches Wörterbuch*, rotates vowels symbols to indicate reduced pronunciation by 180°, or 90° to avoid ambiguity with similar symbols:
- æ → (is rotated as) ᴂ ≠ (and different from) æ
- o → ᴑ ≠ o
- ø → ᴓ ≠ ø
- ɔ → ᴒ ≠ c
- u → ᴝ ≠ n
- ɯ → ᴟ ≠ m
